src/docs/community-docs/CMML/cmml-quickstart.md

```diff
@@ -89,15 +88,13 @@ The sidebar is found on the left side of your ARC.
 
 4. Under __'Wiki'__ you can create or edit a 'Page' for meeting-protocols or writing down sharable notes.
```

Comment:
- writing -> write 
- maybe more pro-active from CMML side: we try to share meeting minutes